src/widgetastic/browser.py

```diff
@@ -392,6 +392,7 @@ def text(self, *args, **kwargs):
             text = self.execute_script(
                 'return arguments[0].textContent || arguments[0].innerText;',
                 self.element(*args, **kwargs))
+            return normalize_space(text) if text is not None else ''
```

Comment:
I think I would prefer doing this here:
```python
if text is None:
    text = ''
```

rather than placing another return in here.